docs/src/main/asciidoc/build-analytics.adoc

```diff
@@ -0,0 +1,166 @@
+
+And here is an example of the data payload being sent:
+
+[source,json]
+.Example analytics payload
+----
+{
+  "context": {
+    "app": {
+      "name": "ZzOlt3q0VgNK0oyUd6NDAhPqJjTSxNOAnqHb5dHfS7M=",
+      "version": "FUDdh16tTKtuJkcoY2Wp57IKKwxZ/Hx42tTvi3Izpmc="
+    },
+    "build": {
+      "gradle_version": "N/A",
+      "maven_version": "3.8.7"
+    },
+    "ci": {
+      "name": "unknown"
+    },
+    "graalvm": {
+      "java_version": "N/A",
+      "vendor": "N/A",
+      "version": "N/A"
+    },
+    "ip": "0.0.0.0",
+    "java": {
+      "vendor": "Eclipse Adoptium",
+      "version": "17.0.4.1"
+    },
+    "kubernetes": {
+      "detected": "false"
+    },
+    "library": {
+      "name": "unknown",
+      "version": "unknown"
+    },
+    "location": {
+      "country": "PT"
+    },
+    "os": {
+      "name": "Mac OS X",
+      "os_arch": "aarch64",
+      "version": "13.3.1"
+    },
+    "quarkus": {
+      "version": "999-SNAPSHOT"
+    },
+    "timezone": "Europe/Lisbon"
+  },
+  "event": "PROD_BUILD",
+  "integrations": {},
+  "messageId": "api-2PCI9LWQM8oRyHIS6fCQMIQ0dlR",
+  "originalTimestamp": "2023-05-01T15:21:22.266033Z",
+  "properties": {
+    "app_extensions": [
+      {
+        "artifact_id": "quarkus-resteasy-jackson",
+        "group_id": "io.quarkus",
+        "version": "999-SNAPSHOT"
+      },
+      {
+        "artifact_id": "quarkus-smallrye-openapi",
+        "group_id": "io.quarkus",
+        "version": "999-SNAPSHOT"
+      },
+      {
+        "artifact_id": "quarkus-hibernate-orm",
+        "group_id": "io.quarkus",
+        "version": "999-SNAPSHOT"
+      },
+      {
+        "artifact_id": "quarkus-opentelemetry",
+        "group_id": "io.quarkus",
+        "version": "999-SNAPSHOT"
+      },
+      {
+        "artifact_id": "quarkus-jdbc-postgresql",
+        "group_id": "io.quarkus",
+        "version": "999-SNAPSHOT"
+      },
+      {
+        "artifact_id": "quarkus-arc",
+        "group_id": "io.quarkus",
+        "version": "999-SNAPSHOT"
+      }
+    ]
+  },
+  "receivedAt": "2023-05-01T15:21:23.184Z",
+  "timestamp": "2023-05-01T15:21:22.266Z",
+  "type": "track",
+  "userId": "872332b4-4b21-42ee-854b-d86c8629e78b"
+}
+----
+
+== Where are the config files?
+Analytics configurations are stored at the `.redhat` folder in the user's home directory:
+
+`anonymousId` is the locally generated anonymous user id.
+
+`com.redhat.devtools.quarkus.localconfig` holds if the user has agreed to send analytics data or not.
+
+`com.redhat.devtools.quarkus.remoteconfig` cache of the remote configuration that allows to disable analytics globally.
+
+== Show me the code!
+
+The bulk of the code is implemented in the `analytics-common` module. Data is sent using calls to `io.quarkus.analytics.AnalyticsService.sendAnalytics(...)` from the Maven and Gradle plugins.
+
+The uploaded events are stored under the project's `/target` folder with the name `build-analytics-event.json`.
+
+== Configuration
```

Comment:
For some users, it might be useful to see a message about the analytics being sent in the app log. ATM, only failed events are logged, but the successfull ones are silent.
Maybe a config property could allow users to make them visible as well.
WDYT?

Reply:
I can add a log line in debug mode. Will that help?

Reply:
Sorry, I meant build log, not app log.

My point is that either in dev mode or in pro build, analytics events are not logged rn. This is nice, as it does not clutter the log. On the other hand, someone might not like it, because the events go unnoticed.

I guess my question is if you think it is worth the effort to make it configurable.

Reply:
it doesn't hurt but given that we publish the sent event on /target and I just made a debug line output, I think it's enough.
We acknowledge when the analytics is activated and it's implicit that will always happen. 
Always publish an info about it might also be annoying.

Reply:
Ok, it's true that there is a clear ack on activation. Let's close this topic then. Thank you.